CHANGELOG_de-DE.md

```diff
@@ -1,5 +1,6 @@
 # 10.1.0
 - Fügt notwenige Zahlungsmittelinformationen zu E-Rechnungen hinzu (shopware/SwagPayPal#255)
+- Behebt ein Problem, bei dem die Formularvalidierung auf der Bestellbestätigungsseite nicht auf das entsprechende Eingabefeld verweist (shopware/SwagPayPal#267)
```

Comment:
this is the changelog of your other PR. It should be something like `Fixes an issue, where the loading spinner is not correctly hidden on invalid confirm order forms`